README.md

```diff
@@ -22,7 +22,7 @@ Currently supported methods of input/output:
 * ElasticSearch {REST API}
 * Redis {format => 'json_event'}
 * RabbitMQ {mechanism => PLAIN}
-* Syslog {format => cee/json, protocol => UDP}
+* Syslog {format => cee/json (based on RFC_5424, not viable for logstash syslog input), protocol => UDP}
```

Comment:
logstash plugins are code-documented and this info is only on https://www.elastic.co/guide/en/logstash/master/plugins-inputs-syslog.html#plugins-inputs-syslog, do you want to include that link or link to code where docstring exists ?